features/registration_feature.feature

```diff
@@ -0,0 +1,27 @@
+Feature: Registration page
+  As a customer
+  In order to login
+  I need to be able to register
+
+  Scenario: Visitor writes his information and presses register
+    Given I visit the register page
+    When I fill in "username" with "Alfred"
+    And I fill in "password" with "whatever"
+    And I fill in "confirm_password" with "whatever"
+    When I press the sign up button
+    Then I would like my information to be saved
+    And I get a message saying "Your user has been created"
+    And I would like to be redirected to the home page
+
+  Scenario: Visitor gets error message when not filling in password
+    Given I visit the register page
+    When I fill in "username" with "Alfred"
```

Comment:
Please use the same format as on your `<lablel>` Isn't it "User name"  or something?

Reply:
Fixed